### PR TITLE
extension synonyms yml and yaml

### PR DESCRIPTION
**Pull request by bogdanvlviv**
Add extension synonyms `yml` and `yaml` for MIME type `application/x-yaml`.

### CODE REVIEW

**Comment by rafaelfranca** [comment]
The bot assigned this to someone else so unless you have a good reason to change please don't change it. As there is no reason to you assign it to me I'll revert it.

File: actionpack/lib/action_dispatch/http/mime_types.rb [hunk 1]
```diff
@@ -21,7 +21,7 @@
 Mime::Type.register "application/xml", :xml, %w( text/xml application/x-xml )
```
Comment (egilburg) [inline actionpack/lib/action_dispatch/http/mime_types.rb]: should `.xml` extension likewise be registered?

also, perhaps refactor `Mime::Type.register` API to use kwargs?

Comment (rafaelfranca) [inline actionpack/lib/action_dispatch/http/mime_types.rb]: `.xml` extension is already registered no?

Comment (egilburg) [inline actionpack/lib/action_dispatch/http/mime_types.rb]: If it is nevermind, just not clear from this line `Mime::Type.register "application/xml", :xml, %w( text/xml application/x-xml )` as it's not passed as the extension argument.

Comment (rafaelfranca) [inline actionpack/lib/action_dispatch/http/mime_types.rb]: you only need to pass the extension argument if you want to create aliases.

**Comment by bogdanvlviv** [comment]
@matthewd, can be merged this pull request or i should close it?

**Pull request merged**